draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -627,14 +632,13 @@ During resumption, the maximum message size is renegotiated.
    follow the sequence for DTLS initialization {{DTLS-init}} and the
    related traffic case.  If a SCTP Server supports DTLS, when
    receiving an INIT chunk with not all DTLS/SCTP Mandatory
-   Options, it can decide to continue by creating an Association with
-   RFC 6083 fallback, plain data only or to ABORT it.
+   Options, it SHOULD reply with ABORT.
```

Comment:
See my comment above. Isn't checking the requirements of an upper layer the job of the upper layer?